File: graphql-dgs-codegen-core/src/integTest/java/com/netflix/graphql/dgs/codegen/java/testcases/dataclasses/dataClassWIthNoFields/expected/types/Person.java
package com.netflix.graphql.dgs.codegen.java.testcases.dataclasses.dataClassWIthNoFields.expected.types;

import java.lang.Override;
import java.lang.String;

public class Person {
  public Person() {
  }

  @Override
  public String toString() {
    return "Person{}";
  }

  public static Builder newBuilder() {
    return new Builder();
  }

  public static class Builder {
    public Person build() {
      Person result = new Person();
      return result;
    }
  }
}

File: graphql-dgs-codegen-core/src/integTest/kotlin/com/netflix/graphql/dgs/codegen/cases/skipCodegenOnInterfaceFields/expected/types/Person.kt
package com.netflix.graphql.dgs.codegen.cases.skipCodegenOnInterfaceFields.expected.types

import com.fasterxml.jackson.`annotation`.JsonTypeInfo
import com.netflix.graphql.dgs.codegen.cases.skipCodegenOnInterfaceFields.expected.Generated
import kotlin.String
import kotlin.Suppress
import kotlin.jvm.JvmName

@Generated
@JsonTypeInfo(
  use = JsonTypeInfo.Id.NAME,
  include = JsonTypeInfo.As.PROPERTY,
  property = "__typename",
)
public sealed interface Person {
  @Suppress("INAPPLICABLE_JVM_NAME")
  @get:JvmName("getName")
  public val name: String?
}
